src/Common/src/Interop/BSD/System.Native/Interop.Sysctl.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using size_t = System.IntPtr;
+
+// This implements shim for sysctl calls.
+// They are available on BSD clones - FreeBSD, OSX and others.
```

Comment:
clone is a controversial term here, it's the original code still in development!

Reply:
I'm fine with avoiding term clone. English is not my strong area . Would 'derivates' or 'descandants' qualify @krytarowski ??

Reply:
`BSD systems`